p2p/event_handler.go

```diff
@@ -9,6 +9,7 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
+// CONSIDER: making this part of some new `ConnManager` concern.
```

Comment:
I think I used the word 'CONSIDERATION`, but `CONSIDER` makes sense too. Just make sure to add it to `TODO_KEYWORDS` in the `Makefile`